### PR TITLE
add eurasianphonology database

### PR DESCRIPTION
**Pull request by kirillkoncha**

### CODE REVIEW

**Comment by bambooforest** [comment]
@kirillkoncha -- but if trying to avoiding ugly, why not remove all the .DS_Store binary files that are added to this PR? :)

**Comment by agricolamz** [comment]
Fair point, @kirillkoncha, I think I've already suggested this in your previous pull request. Remove them all and add to `.gitignore`.

**Comment by agricolamz** [comment]
Dear @kirillkoncha, please remove and I'll merge.

BIN +8 KB tests/testthat/.DS_Store

File: R/eurasianphonology.feature.R [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#' Download data from the database of Eurasian phonological inventories
+#'
+#' This function open downloaded data from the database of Eurasian phonological inventories (\url{http://eurasianphonology.info}) and changes language names to the names from lingtypology database. You need the internet connection.
```
Comment (agricolamz) [inline R/eurasianphonology.feature.R]: opens

Comment (agricolamz) [reply R/eurasianphonology.feature.R]: And are names for real changed? I'd expect them still to be present.

**Comment by agricolamz** [comment]
@kirillkoncha, you also need to change `data/eurasianphonology.RData` since it is a tibble and not a dataframe and contain lists in some columns (`id`, `contr`). Change it to a usual dataframe.

Please rename `contr` to `contributor`.

And don't forget to make appropriate changes in database_creation script.

**Comment by agricolamz** [comment]
@kirillkoncha, you also need to change `data/eurasianphonology.RData` since it is a tibble and not a dataframe and contain lists in some columns (`id`, `contr`). Change it to a usual dataframe.

Please rename `contr` to `contributor`.

And don't forget to make appropriate changes in database_creation script.

It is possible to do e. g. this way:
```
df <- as.data.frame(eurasianphonology)
df$id <- unlist(df$id)
df$language <- unname(df$language)
df$iso <- unname(df$iso)
df$contributor <- unlist(df$contr)
df <- df[,-c(which(names(df) %in% "contr"))]
str(df)
```

File: R/eurasianphonology.feature.R (hunk 1 above)
Comment (kirillkoncha) [reply R/eurasianphonology.feature.R]: There are two columns. One contains the original names from the eurasian phonology database and the other is consisted of changed names.

**Comment by agricolamz** [comment]
Please remove tests/testthat/.DS_Store

**Pull request merged**